### ISSUE TITLE
The `set-output` command is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by matteocorti** [perl-actions/install-with-cpanm#15]
The action uses ```set-output``` which is deprecated.

Example:

```
      - name: install cpanm and Date::Parse
        uses: perl-actions/install-with-cpanm@v1
        with:
          install: Date::Parse
```

produces

```
2023-02-27T12:16:00.2463390Z ##[group]Run perl-actions/install-with-cpanm@v1
2023-02-27T12:16:00.2464360Z with:
2023-02-27T12:16:00.2464590Z   install: Date::Parse
2023-02-27T12:16:00.2464840Z   tests: false
2023-02-27T12:16:00.2465120Z   verbose: false
2023-02-27T12:16:00.2465340Z   sudo: true
2023-02-27T12:16:00.2465710Z   perl: perl
2023-02-27T12:16:00.2466000Z   path: $Config{installsitescript}/cpanm
2023-02-27T12:16:00.2466240Z ##[endgroup]
2023-02-27T12:16:00.4525150Z [command]/usr/local/bin/perl -MConfig -e print "$Config{installsitescript}/cpanm"
2023-02-27T12:16:00.6131190Z /usr/local/Cellar/perl/5.36.0/bin/cpanm::set-output name=Get cpanm from https%3A//cpanmin.us::
2023-02-27T12:16:00.7269540Z ##[warning]The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by BrianCArnold** [comment]
I created PR #16 to fix this, since set-output was being using incorrectly anyway.

Until it's merged, you should be able to replace `perl-action/install-with-cpanm` with `BrianCArnold/install-with-cpanm@stable`

**Comment by atoomic** [comment]
fixed by #16
